mealie/core/config.py

```diff
@@ -18,16 +18,17 @@
 dotenv.load_dotenv(ENV)
 PRODUCTION = os.getenv("PRODUCTION", "True").lower() in ["true", "1"]
 TESTING = os.getenv("TESTING", "False").lower() in ["true", "1"]
+DATA_DIR = os.getenv("DATA_DIR")
 
 
 def determine_data_dir() -> Path:
-    global PRODUCTION, TESTING, BASE_DIR
+    global PRODUCTION, TESTING, BASE_DIR, DATA_DIR
 
     if TESTING:
-        return BASE_DIR.joinpath("tests/.temp")
+        return BASE_DIR.joinpath(DATA_DIR if DATA_DIR else "tests/.temp")
```

Comment:
Is there an instance where you would want to override the testing folder?

Reply:
I dont have a usecase for testing but I guess it does not hurt when you allow to override this for testing too.